images/marketing/pinterest.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What's the reason for providing a static svg and a component? I assume they serve different purposes but I don't see this svg referenced in your PR so just double checking.

Reply:
`data[icon]` in the API is a url to an image. I don't know if it's used by anything – just following what the other plugins do. 🤷 

Will investigate a bit more before merge, someone on Ventures may know more about this.

Reply:
The TLDR is that both are used*.

- The component (in `marketing/components/product-icon`) is used in the `Installed extensions` row in the client/front end.
- The `pinterest.svg` file is used in the API response returned from server in `InstalledExtensions.php`. I don't know if the API field is actually used (*), but this seems consistent with the others (and I don't want to go unpicking it now and add risk). 

We could open an issue to investigate if this API field is actually required and / or used?

Reply:
Thanks for looking into that further. Agreed it makes to be consistent with the others.

I think it's probably more effort than it's worth to investigate if the API field is actually used (due to backwards compatibility concerns). Let's proceed as is!